### DISCUSSION TITLE
GreaterWMS V2.1.26

### DISCUSSION THREAD

**Discussion by Singosgu** [GreaterWMS/GreaterWMS#285] [General]
- Fix some http bug
- release a internal lan version

<hr /><em>This discussion was created from the release <a href='https://github.com/GreaterWMS/GreaterWMS/releases/tag/V2.1.26'>GreaterWMS V2.1.26</a>.</em>

**Comment by sabin451** [comment]
bro can you please contact me i am not able to register admin 
email:sabinsanil19@gmail.com
phone no:9605988449
